### PR TITLE
Full dialog styling

### PR DESCRIPTION
**Pull request by ivannaranjo**
This PR fully styles the "Manage Windows Credentials" dialog have the final look. Adding styles for the `ListBox`, fixing the `Button` styles, font size for all of the text and labels in the UI, etc... 

This PR also adds the banner to the dialog to the `CommonDialogWindowBaseContent` class so all dialogs that need it can get the banner with the right dimensions and layout.

The dialog now looks like this:
screen shot 2016-10-26 at 3 33 45 pm

### CODE REVIEW

**Comment by iantalarico** [comment]
Hey Ivan, I didn't get a chance to get to this today and am doing on-site interviews all day tomorrow.  Is this PR on a time constraint? Okay if I get to it Friday? I can squeeze it in tomorrow if need be.

**Comment by ivannaranjo** [comment]
Friday is OK.

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx [hunk 1]
```diff
@@ -774,7 +774,7 @@
     <comment>The caption or the show credentials button in the dialog.</comment>
   </data>
   <data name="ManageWindowsCredentialsWindowMessage" xml:space="preserve">
-    <value>Windows credentials for {0}</value>
+    <value>Windows _credentials for {0}</value>
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Was this changed by a refactor? Should this just be "credentials"?

File: GoogleCloudExtension/GoogleCloudExtension/ManageWindowsCredentials/ManageWindowsCredentialsWindowContent.xaml [hunk 2]
```diff
@@ -30,13 +38,23 @@
...
             <!-- Caption row -->
-            <TextBlock Margin="3,3,3,2" Text="{Binding InstanceName, StringFormat={x:Static ext:Resources.ManageWindowsCredentialsWindowMessage}}" />
+            <Label Margin="0,0,0,5"
+                   Style="{StaticResource CommonLabelStyle}"
+                   Content="{Binding Message}"
+                   Target="{Binding ElementName=_list}" />
 
             <!-- List of credentials -->
             <ListBox Grid.Row="1"
+                     x:Name="_list"
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/ManageWindowsCredentials/ManageWindowsCredentialsWindowContent.xaml]: why the underscore here?

File: GoogleCloudExtension/GoogleCloudExtension/Theming/ThemingResources.xaml [hunk 3]
```diff
@@ -114,19 +114,24 @@
     </Style>
 
     <!-- Button templates. -->
+    
+    <!-- Margin for the buttons inside of the buttonb ar. -->
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/Theming/ThemingResources.xaml]: s/buttonb ar/button bar

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx (hunk 1 above)
Comment (ivannaranjo) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: The _ adds an access key to the label, after that when the user presses ALT+c the focus will go to the list.

File: GoogleCloudExtension/GoogleCloudExtension/ManageWindowsCredentials/ManageWindowsCredentialsWindowContent.xaml (hunk 2 above)
Comment (ivannaranjo) [inline GoogleCloudExtension/GoogleCloudExtension/ManageWindowsCredentials/ManageWindowsCredentialsWindowContent.xaml]: Names generate fields in the UserControl class, using the `_list` generates then a field that follows our naming conventions.

**Pull request merged**